tests/pyccel/scripts/runtest_multiple_results.py

```diff
@@ -50,8 +50,8 @@ def print_multiple():
 
 print_multiple()
 
-print(min(f3()))
-print(max(f3()))
+# print(min(f3()))
+# print(max(f3()))
```

Comment:
Why is this no longer working?

Reply:
they were linked to numpy min and max.
But I see that builtin min and max, do not behave the same as numpy min, max.
so we need to create a separate class for them.

Reply:
When the argument is a tuple (as here) I think they behave in the same way. Would it be possible to implement that at least so as to not risk breaking code that currently works?
What are the main differences that need to be implemented?

Reply:
builtin max and min can take multiple arguments or an iterator or strings. while np.min or np.max takes only one arg (array).

Reply:
So then could we raise a `NotImplementedError` for all those cases except `iterator` to reproduce the current behaviour?

Reply:
I add now min and max to work like numpy max and min, but they need to handle a lot of cases and create tests for them.
which is out of scope in this PR. 
we need to create an issue for that.